Makefile

```diff
@@ -33,11 +33,14 @@ install-npm-deps:
 	npm install --production
 
 install-npm-deps-dev:
-	npm install --deps
+	npm install
 
 optimize-js: install-npm-deps-dev
 	./node_modules/webpack/bin/webpack.js --config js/webpack.prod.config.js
 
+dev-js: install-npm-deps-dev
+	./node_modules/webpack/bin/webpack.js --config js/webpack.dev.config.js --watch
```

Comment:
Would you mind changing this to `npm run dev`. It essentially does the same, but prevents duplication of this command. While at it, please also change the `optimize-js` to `npm run build`. Thanks!

Reply:
This should be fixed now @ChristophWurst